scripts/skills/racial/rf_zombie_racial.nut

```diff
@@ -47,6 +47,13 @@ this.rf_zombie_racial <- ::inherit("scripts/skills/skill", {
 			});
 		}
 
+		ret.push({
+			id = 12,
+			type = "text",
+			icon = "ui/icons/obituary.png",	// skull icon
```

Comment:
Ok looking at the screenshot now, perhaps the zombie orientation icon would be better?

Reply:
![image](https://github.com/user-attachments/assets/eb03319e-7fd6-4be9-bd07-b095cd6a2ddc)